pkg/commands/engine/object.go

```diff
@@ -111,6 +111,40 @@ The JSON objects can be in either the GenericObjectProperties format or the Gene
 		},
 	}
 
+	var publishObjectCmd = &cobra.Command{
+		Use:     "publish <object-id>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Publish a generic object, like to make a sheet public",
+		Long:    "Publish a generic object, like to make a sheet public",
+		Example: "corectl object publish OBJECT-ID -a APP-ID",
+
+		Run: func(ccmd *cobra.Command, args []string) {
+			commandLineObjects := args[0]
```

Comment:
Suggestion: `objectID` might be a more precise name for this variable.

Reply:
I don't follow, you mean in the example? or instead of args?

Reply:
Instead of `commandLineObjects` you could call the variable `objectID` :)

Reply:
oh got it. I'm just lazy programmer dude. Copy/Paste press any key.